app/models/manageiq/providers/network_manager.rb

```diff
@@ -0,0 +1,20 @@
+module ManageIQ::Providers
+  class NetworkManager < BaseManager
+    class << model_name
+      # define_method(:route_key) { "ems_networks" }
+      # define_method(:singular_route_key) { "ems_network" }
+      # TODO(lsmola) NetworkManager, has to be changed to ems_networks, once we have UI  and Controllers for IT
+      define_method(:route_key) { "ems_clouds" }
+      define_method(:singular_route_key) { "ems_cloud" }
+    end
```

Comment:
@Ladas Can you explain what this is doing?  I don't understand this code at all.